src/pages/WorkloadDetails/WorkloadDetailsPage.tsx

```diff
@@ -13,12 +13,16 @@ import { MetricsObjectTypes } from '../../types/Metrics';
 import CustomMetricsContainer from '../../components/Metrics/CustomMetrics';
 import { serverConfig } from '../../config';
 import BreadcrumbView from '../../components/BreadcrumbView/BreadcrumbView';
+import { GraphType, NodeParamsType, NodeType } from '../../types/Graph';
+import { fetchTrafficDetails } from '../../helpers/TrafficDetailsHelper.tsx';
+import TrafficDetails from '../../components/Metrics/TrafficDetails';
 
 type WorkloadDetailsState = {
   workload: Workload;
   validations: Validations;
   istioEnabled: boolean;
   health?: WorkloadHealth;
+  trafficData: any;
```

Comment:
I share with @jotak my concerns about using any to propagate types.
For something small or very specific it might be ok, but it's hard to read as it is now.
I see this is already discussed.